### PR TITLE
Added Twitch

### PR DESCRIPTION
**Pull request by inquam**

### CODE REVIEW

File: css/zocial.css [hunk 1]
```diff
@@ -134,7 +134,6 @@ a.zocial {
 .zocial.gmail,
 .zocial.instapaper,
 .zocial.itunes,
-.zocial.joinme,
```
Comment (zimbatm) [inline css/zocial.css]: why was joinme removed ?

File: css/zocial.css [hunk 2]
```diff
@@ -119,7 +119,7 @@ a.zocial {
 	background-image: -o-linear-gradient(bottom, rgba(255,255,255,.1), rgba(255,255,255,0) 30%, transparent 50%, rgba(0,0,0,.1));
 	background-image: -webkit-gradient(linear, left top, left bottom, from(rgba(255,255,255,.1)), color-stop(30%, rgba(255,255,255,0)), color-stop(50%, transparent), to(rgba(0,0,0,.1)));
 	background-image: -webkit-linear-gradient(bottom, rgba(255,255,255,.1), rgba(255,255,255,0) 30%, transparent 50%, rgba(0,0,0,.1));
-	background-image: linear-gradient(to bottom, rgba(255,255,255,.1), rgba(255,255,255,0) 30%, transparent 50%, rgba(0,0,0,.1));
+	background-image: linear-gradient(bottom, rgba(255,255,255,.1), rgba(255,255,255,0) 30%, transparent 50%, rgba(0,0,0,.1));
```
Comment (zimbatm) [inline css/zocial.css]: what is this change for ?

Comment (inquam) [inline css/zocial.css]: I followed the exact steps described

1 Install Font Custom
2 Add new font in the src/ folder.
3 Set color settings in the templates/zocial.css file.
4 Run fontcustom compile
5 Update the sample.html file with both the button and icon.
6 Test rendering. If broken go to step 2.
7 Send pull-request !

I only added the twitch line to the templates/zocial.css file, added the .svg to the src folder and ran "fontcustom compile"... So the compilation must have done all that.

File: css/zocial.css (hunk 1 above)
Comment (inquam) [inline css/zocial.css]: I followed the exact steps described

1 Install Font Custom
2 Add new font in the src/ folder.
3 Set color settings in the templates/zocial.css file.
4 Run fontcustom compile
5 Update the sample.html file with both the button and icon.
6 Test rendering. If broken go to step 2.
7 Send pull-request !

I only added the twitch line to the templates/zocial.css file, added the .svg to the src folder and ran "fontcustom compile"... So the compilation must have done all that.

**Comment by zimbatm** [comment]
Thanks for the submission. I've had a look and re-generated the current set and pushed it to master. Do you mind rebasing your changes on top of it ?

**Comment by inquam** [comment]
Is there some quick and easy way to do that on github or do I have to do it using a local checkout? (I'm mostly a SVN guy so getting to know git is a bit of trial and error... hehe)

**Comment by zimbatm** [comment]
oh, sorry for complicating this PR. I'm afraid you'll have to use a local checkout and run `git fetch` to get the latest upsteam objects and then `git rebase origin/master` from your local checkout. You'll probably get some merge conflicts that need to be solved.

Alternatively copy your changes over to a new fresh branch if it's too much hassle.

**Comment by inquam** [comment]
I created a new one... Look at it after the push and it looks much better.

**Pull request closed without merging**